extensions/settings/update.go

```diff
@@ -0,0 +1,21 @@
+package settings
+
+import (
+	v3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	v1 "github.com/rancher/shepherd/clients/rancher/v1"
+)
+
+func EditGlobalSettings(steveclient *v1.Client, globalSetting *v1.SteveAPIObject, value string) (*v1.SteveAPIObject, error) {
```

Comment:
This one needs godoc comments. Another nit; it seems like a `UpdateGlobalSettings`, to unify with the file name.